Ical.Net.Tests/GetOccurrenceTests.cs

```diff
@@ -209,8 +209,29 @@ public void GetOccurrencesShouldEnumerate()
 
         var collection = Calendar.Load(ical);
         var startCheck = new DateTime(2016, 11, 11);
-        var occurrences = collection.GetOccurrences<CalendarEvent>(startCheck, startCheck.AddMonths(1));
-
-        Assert.That(occurrences.Count == 4, Is.True);
+        var occurrences = collection.GetOccurrences<CalendarEvent>(startCheck, startCheck.AddMonths(1))
+            .OrderBy(x => x)
+            .ToList();
+
+        CalDateTime[] expectedStartDates = [
+            new CalDateTime("20161114T000100", "W. Europe Standard Time"),
+            new CalDateTime("20161114T120100", "W. Europe Standard Time"),
+            new CalDateTime("20161121T000100", "W. Europe Standard Time"),
+            new CalDateTime("20161121T120100", "W. Europe Standard Time"),
+            new CalDateTime("20161128T120100", "W. Europe Standard Time"),
+            new CalDateTime("20161128T150100", "W. Europe Standard Time"),
+            new CalDateTime("20161205T000100", "W. Europe Standard Time"),
+            new CalDateTime("20161205T120100", "W. Europe Standard Time"),
```

Comment:
last comma is too much

Reply:
Removed it. Usually I prefer to have it (the comma after the last entry of a list) for a number of reasons, because when adding entries in future changes, Git doesn't detect it as changes, its easier to move lines up and down, and generally looks more consistent. But all that's certainly also a question of taste.